index.html

```diff
@@ -915,12 +915,183 @@ <h4>Get a Specific Presentation</h4>
+workflow interactions, called VC API exchanges, can be given to coordinators.
+      </p>
+      <p>
+A VC API exchange represents a particular interaction based on a given VC API
+workflow. The interaction will take place between an exchange client and the
+workflow service. Exchanges are expected to be transitory, only existing as
+long as the interaction takes to complete. The workflow service stores state
+information about each exchange such as whether the exchange is pending,
+active, or complete, as well as the current step in the workflow, any
+workflow-specific variables and data, and any verifiable presentations and
+credentials received while the exchange executes.
+      </p>
+      <p>
+An issuer, verifier, or holder coordinator is responsible for creating
+exchanges. The coordinator creates an exchange by performing an HTTP POST to
+the `/exchanges` subpath of a chosen workflow, on the workflow service. The
+HTTP request body includes a time-to-live (TTL) for the exchange and any
+variables to be used to populate the workflow's templates for the particular
+exchange. The request body can also include configuration options to enable the
+exchange to be executed using additional protocols beyond VC API. Once the
+exchange is created, an exchange URL that identifies the exchange and enables
+interaction with it is returned to the coordinator.
+      </p>
+      <p>
+The exchange URL is given to the exchange client so that it can initiate the
+exchange. Initiating the exchange does not require any authorization beyond the
+exchange URL. Depending on the workflow service implementation, exchange URLs
+can also be capability URLs (i.e., the URL is an unguessable secret such that
+only whomever is given the URL can initiate the exchange). If the workflow that
+the exchange is based on requires any additional authorization beyond the
+possession of the exchange URL, this is to be obtained during the exchange,
+not at its initiation.
+      </p>
+      <p>
+The exchange URL can also be used by the coordinator to query the current
+state of the exchange as it progresses and to obtain information associated
+with the exchange that the workflow service has stored. Querying the exchange
+in this way requires additional authorization that the coordinator is expected
+to have and that the exchange client is not.
+      </p>
+      <p>
+How the exchange URL is transmitted from a coordinator to an exchange client is
+out of scope for this specification. Known mechanisms for sharing the exchange
+URL with the client include the Credential Handler API (aka CHAPI), a QR
+code, or a universal link.
+      </p>
+      <p>
+VC API exchanges are designed to be executable using other protocols in
+addition to the VC API exchange protocol; for example, an exchange could
+potentially be executable with any of the OID4VCI, OID4VP, DIDComm, and
+VC API exchange protocols. The protocols supported depends on the complexity
```

Comment:
This change (delete `s`) is needed for agreement with the other tweaks by me, @dlongley, and @msporny.
```suggestion
VC API exchange protocols. The protocols supported depend on the complexity
```

Reply:
:( -- your change went in 9 minutes after the PR was merged... can you do this in a new PR (with whatever else you might find)?